string inputString
---
bool status
string outputString
